src/resources/swagger.yaml

```diff
@@ -107,6 +107,31 @@ paths:
           $ref: "#/components/responses/ServerError"
         "200":
           description: "the registration was committed and the donors / specimens / samples added to the clinical db"
+    delete:
+      parameters:
+        - name: id
+          description: the id of the registration
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: programId
+          description: the short name of the program
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        "401":
+          $ref: "#/components/responses/UnauthorizedError"
+        "404":
+          $ref: "#/components/responses/404NotFound"
```

Comment:
you have to add the 404NotFound I can't find it in the swagger components, use https://editor.swagger.io/ to make sure the swagger is valid